designer/new_dialog.py

```diff
@@ -89,6 +91,24 @@ def __init__(self, **kwargs):
         self.list_parent.add_widget(self.listview, 1)
         self.on_adapter_selection_change(self.adapter)
 
+    def _keyboard_closed(self):
+        self._keyboard.unbind(on_key_down=self._on_keyboard_down)
+        self._keyboard = None
+
+    def _on_keyboard_down(self, keyboard, keycode, text, modifiers):
+        '''To detect which key is pressed
+        '''
```

Comment:
check if `modifiers` is empty here, maybe I'm trying to run some shortcut ([ctrl] + up, [ctrl, shift] + down, etc) and these shortcuts cannot interact here.